app/src/androidTest/java/com/github/fribourgsdp/radio/GameActivityTest.kt

```diff
@@ -93,6 +98,8 @@ class GameActivityTest {
         val testIntent = Intent(ctx, GameActivity::class.java).apply {
             putExtra(GAME_IS_HOST_KEY, true)
             putExtra(GAME_KEY, json.encodeToString(fakeGame))
+            putExtra(GAME_UID, 37L)
+            putExtra(IS_IN_TEST_MODE, true)
```

Comment:
As discussed today when you said you were now using mockito, is this still needed?

Reply:
You are right, we will remove it.